{
    "config": {
        "abort": {
            "already_configured": "Urz\u0105dzenie jest ju\u017c skonfigurowane."
        },
        "error": {
            "cannot_connect": "Nie mo\u017cna nawi\u0105za\u0107 po\u0142\u0105czenia, spr\u00f3buj ponownie.",
            "unknown": "Niespodziewany b\u0142\u0105d."
        },
        "step": {
            "user": {
                "data": {
                    "port": "Port szeregowy",
                    "source_1": "Nazwa \u017ar\u00f3d\u0142a #1",
                    "source_2": "Nazwa \u017ar\u00f3d\u0142a #2",
                    "source_3": "Nazwa \u017ar\u00f3d\u0142a #3",
                    "source_4": "Nazwa \u017ar\u00f3d\u0142a #4",
                    "source_5": "Nazwa \u017ar\u00f3d\u0142a #5",
                    "source_6": "Nazwa \u017ar\u00f3d\u0142a #6"
                },
                "title": "Po\u0142\u0105cz z urz\u0105dzeniem"
            }
        }
    }
}